java/ActigraphReader.java

```diff
@@ -72,31 +78,41 @@ public static void readG3TXEpochs(
 
             // underscored are unused for now
             double sampleFreq = -1, accelerationScale = -1, _AccelerationMin, _AccelerationMax;
-            long _LastSampleTime, firstSampleTime=-1;
+            long startDate = -1, stopDate = -1, firstSampleTime=-1;
             String serialNumber = "";
+            String infoTimeZone = "01:00:00"; // default to be UTC time difference
```

Comment:
Shouldn't this be something like `UTC` or `Europe/London` instead of some time offset? A time offset may correspond to many timezones. Also even if it's UTC I guess it should be `"00:00:00"` instead?

Reply:
For `gt3x` files, they call the time shift as timezone because we don't have the timezone information.